ionic-angular-template/src/auth/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,7 @@
+form {
```

Comment:
you can use ionic classes: "ion-padding-start ion-padding-end ion-padding-top ion-padding-bottom" as well as margins and etc. 
Please review: https://ionicframework.com/docs/layout/css-utilities